app/src/main/java/app/revanced/integrations/patches/GeneralAdsPatch.java

```diff
@@ -16,45 +16,45 @@ public final class GeneralAdsPatch extends Filter {
     };
 
     private final BlockRule custom = new CustomBlockRule(
-            SettingsEnum.ADREMOVER_CUSTOM_ENABLED,
-            SettingsEnum.ADREMOVER_CUSTOM_REMOVAL
+            SettingsEnum.ADREMOVER_CUSTOM,
+            SettingsEnum.ADREMOVER_CUSTOM_FILTERS
```

Comment:
You can remove the `ADREMOVER` prefix if we are at it. It was a remnant from Vanceds decompiled code.

Reply:
Currently, we use `HIDE`, `DISABLE` or `ENABLE` prefix. Example:

- `HIDE_GENERAL_ADS`
- `HIDE_PAID_CONTENT`
- `ENABLED_CUSTOM_FILTERS`

Reply:
I don't think sorting is necessarily something we need or ever considered a requirement for this. If we want to group together settings, we should add an explicit way to do so instead of abusing the naming convention. An initial approach could be using categories.